### PR TITLE
fix: Added additional prop

### PR DESCRIPTION
**Pull request by bojangles-m**
GuidelineSearchResultApi: Added additional prop. The prop is need on the web app for analytics

### CODE REVIEW

**Review by joewoodson — APPROVED** [review]

File: packages/app-bridge/src/tests/GuidelineSearchResultApiDummy.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export class GuidelineSearchResultApiDummy {
             section_slug: null,
```
Comment (ragi96) [inline packages/app-bridge/src/tests/GuidelineSearchResultApiDummy.ts:22]: Please add a changeset ;)

Comment (bojangles-m) [reply packages/app-bridge/src/tests/GuidelineSearchResultApiDummy.ts:22]: Added changeset

File: .changeset/tame-bikes-design.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/app-bridge": minor
+---
+
+fix: added additional prop `project_color_id` to the `GuidelineSearchResultApi`.
```
Comment (ragi96) [inline .changeset/tame-bikes-design.md:5]:
```suggestion
feat: added additional prop `project_color_id` to the `GuidelineSearchResultApi`.
```

Is it really a fix? Seems more like a new feature to me

**Review by ragi96 — APPROVED** [review]
changeset lgtm, the rest of the changes I know nothing about it

**Review by olivereisenhut — APPROVED** [review]

**Pull request merged**